platform/bounties.mdx

```diff
@@ -2,4 +2,20 @@
 title: 'Bounties'
 description: 'Found out how you can crowdfund the model'
 icon: 'hand-fist'
----
\ No newline at end of file
+---
+
+Bounty is a core concept of the Replicant Network. It allows users to request and fund the creation of custom AI models. By participating in bounties, users can contribute to the development of models that meet their specific needs while also earning rewards for their contributions.
+
+### Bounty Token
```

Comment:
a lot of this is talked about in the tokenization part? should we have this here or just reference? or just a short description and reference?

Reply:
I'd leave it here since its relevant for this section. If somebody is only interested in bounties but not tokenomics he still learns the jist of it.